src/products/firefox_desktop/adding_pontoon.md

```diff
@@ -1,7 +1,6 @@
 # Adding a new locale to Pontoon
 
 * If the locale is not [available](https://pontoon.mozilla.org/teams/) in Pontoon yet, [consult this document](../../tools/pontoon/adding_new_locale.md) for instructions on how to add it.
-* File a bug to create a new Mercurial repository in [l10n-central](https://hg.mozilla.org/l10n-central/) using [this bug template](https://mzl.la/3VxIZgs) (replace `%LOCALE_CODE%` with the actual locale code).
-* Push at least one change to the repository, otherwise Pontoon will not pick it up during sync. The safest change is to initialize the `toolkit/chrome/global/intl.properties` file with the correct values ([example](https://hg.mozilla.org/l10n-central/ppl/rev/b3fd0faf59b0b45b2cf30c01d85157beee2a0bd0)).
+* Create a subfolder using the locale code (e.g. `/skr`) in ([firefox-l10n](https://github.com/mozilla-l10n/firefox-l10n)) and push at least one change to the repository, otherwise Pontoon will not pick it up during sync. The safest change is to initialize the `toolkit/chrome/global/intl.properties` file with the correct values ([example](https://github.com/mozilla-l10n/firefox-l10n/commit/a887bff12994779ea8e605f4abc1b240a287ccb9)).
```

Comment:
There is no need to create a folder, Pontoon already knows the repository. The idea of initializing `intl.properties` is still valid though.